system-tests/e2e-transfer-test/runner/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+plugins {
+    java
+}
+
+val jupiterVersion: String by project
+val restAssured: String by project
+
+dependencies {
+    testImplementation(testFixtures(project(":common:util")))
+    testImplementation(testFixtures(project(":launchers:junit")))
+
+    testImplementation("io.rest-assured:rest-assured:${restAssured}")
+    testImplementation("org.assertj:assertj-core:3.21.0")
+    testImplementation("org.awaitility:awaitility:4.1.1")
```

Comment:
Do we have some sort of convention or similar describing how we want to define the versions of dependencies? I'm wondering if it wouldn't make more sense to manage all version numbers from one place (`/gradle.properties`) for the whole project.

For `awaitility` we already defined something:
<img width="725" alt="image" src="https://user-images.githubusercontent.com/97295414/156254379-b022eccc-bbf3-45b2-b99f-a9603a98c522.png">